src/Blocks/components/heading/heading-style.scss

```diff
@@ -14,6 +14,10 @@ $heading: (
 				font-size: 54px,
 				line-height: 1,
 			),
+			summary: (
```

Comment:
summary for the font size is kinda odd please use something else that represents sizes. Please check other projects

Reply:
Is `small` ok?